src/Service/Sns/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## NOT RELEASED
 
+### Added
```

Comment:
This should be "fixed" as it does not require a new minor release.

Reply:
:-( sorry about that

Reply:
No worries. We will fix in when we prepare for a release next time.